shared/rakefiles/entrypoint_common.rake

```diff
@@ -7,12 +7,12 @@ task :apply_common_infra => [:set_vars, :configure_aws_restore] do
   sh "#{@exekube_cmd} rake apply_common_infra"
 end
 
-desc "[ONLY ADMIN] Fix the permissions for the common service account"
-task :fix_common_service_account_permissions => [:set_vars] do
-  # This task sets the permissions of the common service account for creating the
+desc "[ONLY ADMIN] Fix the permissions for admin accounts in the organization"
+task :fix_organization_permissions => [:set_vars] do
```

Comment:
What about calling this task `setup_...`? Setup to me sounds like something that you do initially, fix sounds more like something you do to repair something broken

Reply:
mmm, well it's the command that rake proposes you when it detects that the permissions are wrong in the IAM organization (fix). Also, is a step that you have to do initially when a new organization is created (setup).

Either way works for me :dancing_men: